data_steward/gcloud/gcs/__init__.py

```diff
@@ -106,6 +106,7 @@ def get_hpo_bucket(self, hpo_id: str) -> Bucket:
 
         try:
             bucket = self.bucket(bucket_name)
+            self.get_bucket_items_metadata(bucket)
```

Comment:
Add in a comment here saying this is to verify the bucket can be accessed and we don't care about the result.